### PR TITLE
Windows: Remove DNS module and update README

### PR DESCRIPTION
**Pull request by keith-smiley-gravwell**
No issues addressed here just need to remove DNS module

### CODE REVIEW

File: windows/searchlibrary/0f7750bd-1576-481a-b60d-b382bc9e70fd.query [hunk 1]
```diff
@@ -6,10 +6,10 @@
 tag=$WINDOWS_LOGON winlog TimeCreated EventID == 4624 Computer SubjectUserName SubjectDomainName TargetUserName TargetDomainName LogonType LogonProcessName WorkstationName as TargetInfo IpAddress
 | lookup -s -r @groups [Computer SubjectUserName SubjectDomainName TargetUserName TargetDomainName LogonType] [Comp SUN SDN TUN TDN LT]
 | lookup -r windows_login_types LogonType logon_type name as LogonName
-| dns -reverse IpAddress Host
-| printf -e SourceInfo "%v (%v)" Host IpAddress
+/* | dns -reverse IpAddress Host */
```
Comment (kris-watts-gravwell) [inline windows/searchlibrary/0f7750bd-1576-481a-b60d-b382bc9e70fd.query:9]: don't comment it out, just remove it

File: windows/searchlibrary/0f7750bd-1576-481a-b60d-b382bc9e70fd.query [hunk 2]
```diff
@@ -6,10 +6,10 @@
 tag=$WINDOWS_LOGON winlog TimeCreated EventID == 4624 Computer SubjectUserName SubjectDomainName TargetUserName TargetDomainName LogonType LogonProcessName WorkstationName as TargetInfo IpAddress
 | lookup -s -r @groups [Computer SubjectUserName SubjectDomainName TargetUserName TargetDomainName LogonType] [Comp SUN SDN TUN TDN LT]
 | lookup -r windows_login_types LogonType logon_type name as LogonName
-| dns -reverse IpAddress Host
-| printf -e SourceInfo "%v (%v)" Host IpAddress
+/* | dns -reverse IpAddress Host */
+| printf -e SourceInfo "%v" IpAddress
```
Comment (kris-watts-gravwell) [inline windows/searchlibrary/0f7750bd-1576-481a-b60d-b382bc9e70fd.query:10]: just use `IpAddress` directly in the query, don't format it as a new name, this is just load and achieves nothing.

File: windows/searchlibrary/0f7750bd-1576-481a-b60d-b382bc9e70fd.query [hunk 3]
```diff
@@ -6,10 +6,10 @@
...
-| printf -e SourceInfo "%v (%v)" Host IpAddress
+/* | dns -reverse IpAddress Host */
+| printf -e SourceInfo "%v" IpAddress
 | printf -e LogonTypeInfo "%v (%v)" LogonName LogonType
 | printf -e SubjectUser "%v\\%v" SubjectDomainName SubjectUserName
 | printf -e TargetUser "%v\\%v" TargetDomainName TargetUserName
 | eval (if (TargetInfo == "-") {TargetInfo = "<See Computer>";})
-| table TimeCreated Computer SubjectUser TargetUser LogonTypeInfo LogonProcessName TargetInfo SourceInfo
\ No newline at end of file
+| table TimeCreated Computer SubjectUser TargetUser LogonTypeInfo LogonProcessName TargetInfo SourceInfo
```
Comment (kris-watts-gravwell) [inline windows/searchlibrary/0f7750bd-1576-481a-b60d-b382bc9e70fd.query:15]: use `IpAddress` not `Source Info`

File: windows/searchlibrary/357db0e6-7346-40af-8863-f3468f3ae991.query [hunk 4]
```diff
@@ -4,11 +4,11 @@ tag=$WINDOWS_LOGON winlog EventID == 4624 Computer SubjectUserName SubjectDomain
     !(SubjectUserName ~ "$" && (TargetUserSid !~ "S-1-5-80" || TargetUserSid ~ "S-1-5-82" || TargetUserSid ~ "S-1-5-90" || TargetUserSid ~ "S-1-5-96")) 
     || !((SubjectUserName == "-" || SubjectUserName ~ "$") && TargetUserSid == "S-1-5-18") )
 | lookup -r windows_login_types LogonType logon_type name as LogonName
-| dns -reverse IpAddress Host
-| printf -e SourceInfo "%v (%v)" Host IpAddress
+/* | dns -reverse IpAddress Host */
+| printf -e SourceInfo "%v" IpAddress
```
Comment (kris-watts-gravwell) [inline windows/searchlibrary/357db0e6-7346-40af-8863-f3468f3ae991.query:8]: same

File: windows/searchlibrary/59b0e472-3f9a-42c0-982d-3126923b97ae.query [hunk 5]
```diff
@@ -2,10 +2,10 @@ tag=$WINDOWS_LOGON winlog TimeCreated Computer EventID SubjectUserSid SubjectDom
 | eval (EventID=="4624" || EventID=="4625")
 | eval ((SubjectUserSid=="S-1-0-0" && LogonType=="3" && LogonProcessName=="NtLmSsp") || (LogonType=="9" && LogonProcessName=="seclogo"))
 | lookup -r windows_login_types LogonType logon_type name as LogonName
-| dns -reverse IpAddress Host
-| eval if (Host~"localhost") Host="localhost.";
-| unique TimeCreated Computer TargetDomainName TargetUserName Host IpAddress LogonName LogonType LogonProcessName
+/* | dns -reverse IpAddress Host
```
Comment (kris-watts-gravwell) [inline windows/searchlibrary/59b0e472-3f9a-42c0-982d-3126923b97ae.query:5]: same

File: windows/searchlibrary/59b0e472-3f9a-42c0-982d-3126923b97ae.query [hunk 6]
```diff
@@ -2,10 +2,10 @@ tag=$WINDOWS_LOGON winlog TimeCreated Computer EventID SubjectUserSid SubjectDom
...
-| unique TimeCreated Computer TargetDomainName TargetUserName Host IpAddress LogonName LogonType LogonProcessName
+/* | dns -reverse IpAddress Host
+| eval if (Host~"localhost") Host="localhost."; */
+| unique TimeCreated Computer TargetDomainName TargetUserName IpAddress LogonName LogonType LogonProcessName
 | printf -e TargetUser "%v\\%v" TargetDomainName TargetUserName
 | printf -e LogonTypeInfo "%v (%v)" LogonName LogonType
-| printf -e HostInfo "%v (%v)" Host IpAddress
-| table TimeCreated Computer TargetUser HostInfo LogonTypeInfo LogonProcessName
\ No newline at end of file
+| printf -e HostInfo "%v" IpAddress
```
Comment (kris-watts-gravwell) [inline windows/searchlibrary/59b0e472-3f9a-42c0-982d-3126923b97ae.query:10]: same

**Review by kris-watts-gravwell — CHANGES_REQUESTED** [review]
don't comment out the op, this isn't something we want to teach users in a generic kit like this.

Also, don't format the IP address, all the rendering stuff already does that, just use `IpAddress` directly

File: windows/searchlibrary/0f7750bd-1576-481a-b60d-b382bc9e70fd.query [hunk 7]
```diff
@@ -6,10 +6,7 @@
 tag=$WINDOWS_LOGON winlog TimeCreated EventID == 4624 Computer SubjectUserName SubjectDomainName TargetUserName TargetDomainName LogonType LogonProcessName WorkstationName as TargetInfo IpAddress
 | lookup -s -r @groups [Computer SubjectUserName SubjectDomainName TargetUserName TargetDomainName LogonType] [Comp SUN SDN TUN TDN LT]
 | lookup -r windows_login_types LogonType logon_type name as LogonName
-| dns -reverse IpAddress Host
-| printf -e SourceInfo "%v (%v)" Host IpAddress
-| printf -e LogonTypeInfo "%v (%v)" LogonName LogonType
 | printf -e SubjectUser "%v\\%v" SubjectDomainName SubjectUserName
```
Comment (kris-watts-gravwell) [inline windows/searchlibrary/0f7750bd-1576-481a-b60d-b382bc9e70fd.query:9]: you deleted the LogonTypeInfo, this query will fail to parse

**Review by kris-watts-gravwell — CHANGES_REQUESTED** [review]
i think you deleted one line too many on one query

File: windows/searchlibrary/0f7750bd-1576-481a-b60d-b382bc9e70fd.query (hunk 7 above)
Comment (keith-smiley-gravwell) [reply windows/searchlibrary/0f7750bd-1576-481a-b60d-b382bc9e70fd.query:9]: Yeah caught that...already fixing

**Review by kris-watts-gravwell — CHANGES_REQUESTED** [review]
Bump version on changes please

**Review by mike-wade-gravwell — APPROVED** [review]
LGTM

**Review by kris-watts-gravwell — APPROVED** [review]
LGTM

**Pull request merged**